lib/libc/zoneinfo/Makefile

```diff
@@ -102,7 +102,7 @@ tzdb:
 
 .tzbuilt: tzbin .tzunpack
 	$(Q) $(MAKE) -C tzdb CFLAGS=-DSTD_INSPIRED TOPDIR=$(TZBIN_PATH) posix_only
-ifeq ($(CONFIG_LIBC_DELETE_ZONEINFO_RULES),y)
+ifeq ($(CONFIG_LIBC_OPTIMIZE_ZONEINFO_RULES),y)
```

Comment:
It is better to inform to us where or how it is defined. 
The 'CONFIG_LIBC_OPTIMIZE_ZONEINFO_RULES' was defined in [TizenRT]/lib/libc/Kconfig.

Reply:
I don't think it is necessary to describe where it is defined in here.